docs/plugins/protocol-http.md

```diff
@@ -27,6 +27,8 @@ Uses `node-fetch` library.
 - `timeout`: number - timeout in ms
 - `withCredentials`: boolean - configure credentials type, [see](#request-credentials)
 - `abortPromise`: Promise - if passed promise resolves, request get aborted with the result of promise
+- `signal`: AbortSignal - signal used to abort current request
+- `requestOptions` RequestInit - any additional options passed directly to window.fetch or node-fetch instance
```

Comment:
Maybe `fetchOptions` will be better?
And I suggest to exclude signal, timeout, method and others from this property interface.

Reply:
I've removed this commit as we decided to not implement such functionality